addons/fieldmanual/CfgHints.hpp

```diff
@@ -0,0 +1,391 @@
+#include "script_component.hpp"
```

Comment:
```suggestion
```

Reply:
not sure what you are suggesting here. do you not want macros like CSTRING in the config? Or should I just add that particular macro that I use?

Reply:
the `#include "script_component.hpp"` is done in config.cpp

config.cpp includes CfgHint.hpp, so there's no need to include it a 2nd time

Reply:
Oh derp